lib/providers/reddit.js

```diff
@@ -0,0 +1,29 @@
+// Load modules
+
+
+// Declare internals
+
+var internals = {};
+
+
+exports = module.exports = function (options) {
+    return {
```

Comment:
I need to add our lint rules... but we have an extra newline after every function declaration.

Please add a newline here and on line 17